src/api/mapping.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express'
+import uuid4 from 'uuid/v4'
+const mappingRouter = express.Router()
+const list =[]
+
+mappingRouter.post('/mappings', (req, res) =>{
+  const domainObject = {
+    'domain': req.body.domain,
+    'port' : req.body.port,
+    'ip' : req.body.ip,
+    'id' : uuid4()
+  }
+  list.push(domainObject)
```

Comment:
don't use list, refer to this to see how `serviceKeys` is handling the `post` requests: https://github.com/garageScript/myProxy/blob/master/src/api/admin.ts#L37